.azure-pipelines-templates/matrix.yml

```diff
@@ -35,14 +35,9 @@ parameters:
       cmake_args: ''
     debug:
       cmake_args: '-DCMAKE_BUILD_TYPE=Debug -DBUILD_SMALLBANK=OFF'
-    perf:
-      HTTP:
-        cmake_args: ''
-    release:
-      FTCP:
-        cmake_args: '-DCMAKE_INSTALL_PREFIX=./install'
-      HTTP:
-        cmake_args: '-DCMAKE_INSTALL_PREFIX=./install'
+    install:
+      install_prefix: '/tmp/ccf-install'
+      cmake_args: '-DCMAKE_INSTALL_PREFIX=/tmp/ccf-install'
```

Comment:
I guess we cannot do `cmake_args: '-DCMAKE_INSTALL_PREFIX=${{ build.install.install_prefix}}'` ?

Reply:
No: https://dev.azure.com/MSRC-CCF/CCF/_build/results?buildId=4648&view=results

Reply:
I'd prefer to just have the `install_prefix`, remove `install: true`, and then in `common.yml` append the cmake_args with something like

`${{ if(parameters.install_prefix, '-DCMAKE_INSTALL_PREFIX=${{ parameters.install_prefix }}', '' }}`

but it seems this is not possible.